src/main/java/org/folio/entlinks/service/messaging/authority/handler/DeleteAuthorityChangeHandler.java

```diff
@@ -42,11 +44,17 @@ public List<LinksChangeEvent> handle(List<AuthorityChangeHolder> changes) {
         .toList();
 
     var authorityIds = linksEvents.stream().map(LinksChangeEvent::getAuthorityId).collect(Collectors.toSet());
-    var softDeleteAuthorityIds = changes.stream().map(AuthorityChangeHolder::getAuthorityId).toList();
+    var softDeleteAuthorityIds = changes.stream()
+        .filter(change -> change.getChangeType().equals(DELETE))
```

Comment:
Why do we filter by `DELETE` change type here if handler should be called only for `supportedAuthorityChangeType` which is `DELETE` for this handler?

Reply:
Yes, but this delete handler can be called for ```UPDATE``` event as well, you can have a loot at method ```InstanceAuthorityLinkUpdateService#processEventsByChangeType``` and ```change.getChangeType()``` returns not just change holder's type, but has a logic inside - I have covered this also in the added unit test